presto-docs/src/main/sphinx/optimizer.rst

```diff
@@ -8,3 +8,4 @@ Query Optimizer
     optimizer/statistics
     optimizer/cost-in-explain
     optimizer/cost-based-optimizations
+    optimizer/caching
```

Comment:
This doesn't belong to query optimizer. Maybe just create a top-level catalog "Data Cache" and make this `cache/alluxio`